### PR TITLE
4.2.1 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 4.2.1 (2020-12-02)
+
+- Fixed the colorpicker component button to show proper hover and focus styles. [#77](https://github.com/blackbaud/skyux-colorpicker/pull/77)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Fixed the hover and focus styles for the colorpicker button. [#77](https://github.com/blackbaud/skyux-colorpicker/pull/77)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Fixed the colorpicker component to properly position the picker below the trigger button. [#79](https://github.com/blackbaud/skyux-colorpicker/pull/79)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:4]:
```suggestion
- Fixed the colorpicker component to position the picker below the trigger button. [#79](https://github.com/blackbaud/skyux-colorpicker/pull/79)
```

**Review by blackbaud-johnly — APPROVED** [review]
Minor edits suggested. But approving in case you prefer to merge as-is.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-4.2.1-release](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-4.2.1-release)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-4.2.1-release](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-4.2.1-release)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-4.2.1-release](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-4.2.1-release)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**